config/.env.template

```diff
@@ -3,6 +3,7 @@ POLL_PERIOD_SECOND=300
 SUBGRAPH_ENDPOINT='https://api.thegraph.com/subgraphs/name/aminlatifi/givpower-deployment-six'
 GIVPOWER_CONTRACT_ADDRESS='0x898Baa558A401e59Cb2aA77bb8b2D89978Cf506F'
 PRIVATE_KEY=''
-UNLOCK_PER_TRANSACTION = 10
-GAS_MAX_BASE_FEE: 1.5;
-GAS_PRIORITY_FEE: 1.5;
+UNLOCK_PER_TRANSACTION=10
+GAS_MAX_BASE_FEE=1.5
+GAS_PRIORITY_FEE=1.5
+NODE_ENV=develop
```

Comment:
There is no need to set `NODE_ENV`  here, the env file is already chosen by value of this env var already

Reply:
> There is no need to set `NODE_ENV` here, the env file is already chosen by value of this env var already

True, but the idea is that in this way, the docker-compose file can be as clean as possible and not dependent on what ENV values are put inside it. And we throw all the env configs to the .env file